next.js/src/app/MyUserPage/components/ProductCard.tsx

```diff
@@ -0,0 +1,49 @@
+import Image from "next/image";
+
+type ProductProps = {
+  product_name: string;
+  id: number;
+  model_number: string;
+  price: number;
+  description: string;
+  company_name: string;
+  priority: boolean;
+};
+
+export const ProductCard: React.FC<ProductProps> = ({
+  id,
+  product_name,
+  description,
+  company_name,
+  price,
+  model_number,
+  priority = false, //1件目のみtrueで受け取るためデフォルトはfalse
+}) => {
```

Comment:
![image](https://user-images.githubusercontent.com/50566499/259609094-fc0e8c2f-5f88-4927-8bb1-7d160c2279c9.png) 
考え方いろいろありますが、 `React.FC` より以下のようにすることが多いイメージです！

```suggestion
export const ProductCard = ({
  id,
  product_name,
  description,
  company_name,
  price,
  model_number,
  priority = false, //1件目のみtrueで受け取るためデフォルトはfalse
}: ProductProps) => {
```

Reply:
こっちでいけたんですね💦
時々今回みたいに設定してました。。。

ありがとうございます🙇‍♂️